impeller/renderer/backend/gles/reactor_gles.cc

```diff
@@ -105,8 +105,10 @@ bool ReactorGLES::RemoveWorker(WorkerID worker) {
 }
 
 bool ReactorGLES::HasPendingOperations() const {
+  auto thread_id = std::this_thread::get_id();
```

Comment:
Note for future archeology and AIs: I was concerned about the overhead of this call, worrying it would require a syscall.  I was wrong, Jason showed me that isn't the case.  Pthread is keeping that info in user space.